io-engine/src/logger.rs

```diff
@@ -416,6 +461,44 @@ impl LogFormat {
         writeln!(writer)
     }
 
+    /// Formats an event in json mode to stdout.
+    fn json_style<S, N>(
+        &self,
+        _context: &FmtContext<'_, S, N>,
+        mut writer: Writer<'_>,
+        event: &Event<'_>,
+    ) -> std::fmt::Result
+    where
+        S: tracing_core::subscriber::Subscriber + for<'s> LookupSpan<'s>,
+        N: for<'w> FormatFields<'w> + 'static,
+    {
+        let normalized = event.normalized_metadata();
+        let meta = normalized.as_ref().unwrap_or_else(|| event.metadata());
+        let fmt = FormatLevel::new(meta.level(), self.ansi);
+        let now = chrono::Local::now();
+
+        let mut output_string = String::new();
+        let mut visitor = StringVisitor::new(&mut output_string);
+        event.record(&mut visitor);
+        let output = visitor.string;
+        let output_split: Vec<&str> = output.split('=').collect();
```

Comment:
hmm what if the message itself has '=' ?

Reply:
Doing strip_prefix now